docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+# As of 9-12-17, Amazon ECS CLI supports Docker compose file syntax versions
+# 1 and 2
+# (see: http://docs.aws.amazon.com/AmazonECS/latest/developerguide/cmd-ecs-cli-compose.html)
+version: '2'
+
+services:
+
+  elasticsearch:
+    # *Note: We are using Elasticsearch version 2.x because django-haystack,
+    # one of our dependencies, only supports Elasticsearch versions
+    # 1.x and 2.x, not the newer versions (as of 8/7/17). See:
+    # http://django-haystack.readthedocs.io/en/v2.6.0/installing_search_engines.html#elasticsearch
+    #
+    # Also, we are using Elasticsearch v2.3 (even though v2.4.6 is
+    # available) because it is the highest version that AWS supports of the
+    # 2.x major version (as of 8/7/17). The logic here is that we want the
+    # local deployment to mirror our deployment on AWS. See:
+    # https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-gsg.html
+    image: elasticsearch:2.3
```

Comment:
I think this version is okay if it works for us, but I want to clarify: the AWS support note above does not apply to us since we are rolling our own via Docker here.

I guess if we wanted to we could deploy to the AWS-supplied version of Elasticsearch so it may be wise to keep what we use in sync. I don't quite see a reason to "undo" our current AWS independence, however, unless we happened to run into some issue with search performance or something.